src/dachshund/transformer.rs

```diff
@@ -281,6 +280,11 @@ impl Transformer {
         output: &Sender<(String, bool)>,
     ) -> CLQResult<Option<BeamSearchResult<'a, TGraph>>> {
         if graph.get_core_ids().is_empty() || graph.get_non_core_ids().unwrap().is_empty() {
+            // still have to send an acknowledgement to the output channel
+            // that we have actually processed this graph, otherwise
+            // we lose track of how many graphs have been processed so
+            // far!
+            output.send(("".to_string(), false)).unwrap();
```

Comment:
Can you change this so that it doesn't create empty lines in the output? I think the simplest way is to change this line to 

output.send((None, false)).unwrap();

(and make the corresponding type change output: &Sender<(Option<String>, bool)>).